core/models.py

```diff
@@ -385,7 +385,10 @@ def is_staff(self):
 
     @property
     def is_superuser(self):
-        return False
+        if isinstance(self, TechnicalUser):
```

Comment:
This statement is never true, interactive user is never instance of technical user. You have to check User class (but now user doesn't have related_name, so please add it as well) And then use something like self.user.is_superuser()